pkg/gotohelm/transpiler.go

```diff
@@ -1051,6 +1051,8 @@ func (t *Transpiler) transpileCallExpr(n *ast.CallExpr) Node {
 		return &Selector{Expr: args[0], Field: "AsMap"}
 	case "github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette.UnmarshalInto":
 		return args[0]
+	case "github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette.MustDecodeYamlInto":
```

Comment:
Left over change? I don't see `MustDecodeYamlInto` anywhere.

Reply:
yeah, this was briefly a thing. Per discussion yesterday, we'll push on with changing the values schema here.